### PR TITLE
[STAKAN-124]: Hide currency icons + Add AVAX icon

### PR DESCRIPTION
**Pull request by Kud8**
Hello, brave developer, nice day to merge a pull request, ain't it?

## Follow the checklist to proceed

- [x] I've performed a code self-review
- [x] I've tested feature or bugfix on my own
- [x] If I make changes to another person's module, I've asked how to use it or request a review
- [x] I've checked mobile & tablet screens (if have changes in UI)
- [x] Local build successfully completed
- [x] Branch name and pull request title are according to [the requirements](https://ankrnetwork.atlassian.net/wiki/spaces/STAKAN/pages/1666646162/Branches+and+pull+request+naming)

### CODE REVIEW

File: src/modules/layout/components/WalletBalance/WalletBalance.tsx [hunk 1]
```diff
@@ -4,42 +4,54 @@ import classNames from 'classnames';
...
   bnbBalance,
   aeth,
+  avax,
   className,
+  blockchainType,
 }: IWalletBalanceProps) => {
   const classes = useWalletBalance();
+  const showMainBalances =
+    blockchainType === Blockchain.ethereum ||
+    blockchainType === Blockchain.binance;
```
Comment (AndrewKhizhnyak) [inline src/modules/layout/components/WalletBalance/WalletBalance.tsx:31]:
```suggestion
  const showMainBalances =
    [Blockchain.ethereum, Blockchain.binance].includes(blockchainType);
```

File: src/modules/layout/components/WalletBalance/WalletBalance.tsx [hunk 2]
```diff
@@ -4,42 +4,54 @@ import classNames from 'classnames';
...
-          {aeth ? aeth.decimalPlaces(DEFAULT_FIXED).toFormat() : 0}
-        </Typography>
+        {showMainBalances && (
+          <>
+            <Typography
+              color="textSecondary"
+              className={classNames(classes.item, classes.aeth)}
+              title="aETHc"
+            >
+              {aeth ? aeth.decimalPlaces(DEFAULT_FIXED).toFormat() : 0}
```
Comment (AndrewKhizhnyak) [inline src/modules/layout/components/WalletBalance/WalletBalance.tsx:43]:
```suggestion
              {aeth?.decimalPlaces(DEFAULT_FIXED).toFormat() ?? 0}
```

File: src/modules/layout/components/WalletBalance/WalletBalance.tsx [hunk 3]
```diff
@@ -4,42 +4,54 @@ import classNames from 'classnames';
...
-          title="Ethereum"
-        >
-          {ethereum ? ethereum.decimalPlaces(DEFAULT_FIXED).toFormat() : 0}
-        </Typography>
+            <Typography
+              color="textSecondary"
+              className={classNames(classes.item, classes.ethereum)}
+              title="Ethereum"
+            >
+              {ethereum ? ethereum.decimalPlaces(DEFAULT_FIXED).toFormat() : 0}
```
Comment (AndrewKhizhnyak) [inline src/modules/layout/components/WalletBalance/WalletBalance.tsx:51]: same https://github.com/Ankr-network/stkr-web3/pull/281/files#r689805251

**Review by AndrewKhizhnyak — APPROVED** [review]

File: src/modules/layout/components/WalletBalance/WalletBalance.tsx [hunk 4]
```diff
@@ -60,6 +72,16 @@ export const WalletBalance = ({
             {ankr.decimalPlaces(DEFAULT_FIXED).toFormat()}
           </Typography>
         )}
+
+        {avax && (
+          <Typography
+            color="textSecondary"
+            className={classNames(classes.item, classes.avax)}
+            title="Avax"
```
Comment (BigBadAlien) [inline src/modules/layout/components/WalletBalance/WalletBalance.tsx:80]:
```suggestion
            title={t('foobar.avax')}
```

All literals must be translated 🤷‍♂️

**Review by BigBadAlien — APPROVED** [review]

File: src/modules/layout/components/WalletBalance/WalletBalance.tsx (hunk 2 above)
Comment (BigBadAlien) [reply src/modules/layout/components/WalletBalance/WalletBalance.tsx:43]: @Kud8 Btw. Zero shouldn't be shown if undefined. Let's bear in mind in the upcoming version

**Pull request merged**